### DISCUSSION TITLE
Release note generator: pipeline variable in a Handlebar  template

### DISCUSSION THREAD

**Discussion by Panda-Sharp** [rfennell/AzurePipelines#1222] [Q&A] [answered]
Hello,

Is there any way to print a pipeline variable (eg: $(platform)) into a Handlebar Templates?
I gave it a  random shot with {{platform}} but doesn't work.

Thanks!

**Comment by rfennell** [comment, marked as answer]
Have a look at #756

But if you use an inline handlebars template you can use `$(var)` in the template and it will be expanded before the template is used

**Reply by Panda-Sharp** [reply]
> Have a look at #756
> 
> But if you use an inline handlebars template you can use `$(var)` in the template and it will be expanded before the template is used
> 

Unfortunately we're using file templates, but thanks for your feedbacks, I got few ideas.

**Reply by rfennell** [reply]
The other solution I know people have used is to put some tag/marker of your own in the template e.g. `##mytag##` and the use an extra task/script do a tag replace with the `$(var)` predefined variables

**Reply by Panda-Sharp** [reply]
> The other solution I know people have used is to put some tag/marker of your own in the template e.g. `##mytag##` and the use an extra task/script do a tag replace with the `$(var)` predefined variables

yeah that's what I was thinking to do. thanks again!
